dt-mapping/mapping-queries.php

```diff
@@ -1301,34 +1301,72 @@ public static function post_type_geojson( $post_type, $args = [], $offset = 0, $
         global $wpdb;
 
         //phpcs:disable
+        // Determine if post id filtering should take place.
+        $post_id_filter_sql = '';
+        $shared_user_join_sql = '';
+        $shared_user_condition_sql = '';
+        if ( isset( $args['list_all'], $args['list_all_by_user_id'] ) && !$args['list_all'] ) {
```

Comment:
`list_all` seems to be set by the browser.
Consider checking if the user can actually list all. the new genmapper should have code to copy.